src/manifest.c

```diff
@@ -658,6 +672,10 @@ char *file_type_to_string(struct file *file)
 		type[1] = 'd';
 	}
 
+	if (file->is_ghosted) {
```

Comment:
Switch statement instead?

Reply:
Doesn't work as well going this direction (checking a bunch of different `int` fields and assigning `char`s). Anyways that's a restructure of the function and I'm just adding this single conditional so I'd call that out-of-scope for this change.